prover/protocol/compiler/globalcs/quotient.go

```diff
@@ -258,10 +258,14 @@ func (ctx *quotientCtx) Run(run *wizard.ProverRuntime) {
 			// normal case for interleaved or repeated columns
 			witness := pol.GetColAssignment(run)
 
-			witness = sv.FFTInverse(witness, fft.DIF, false, 0, 0, nil)
+			// The retry is because we noticed some panics in the FFT and we
+			// could not figure out why but retrying works. We also could not
+			// reproduce it as it was random and rare.
+			utils.RetryIfPanic(func() {
```

Comment:
I don't think this is useful; the error in the log if I recall is a SEGFAULT, so it will not trigger a panic

Reply:
Yeah, just tested and it will not work